### ISSUE TITLE
Grid Pattern: Missing guidance for use of aria-selected to support column selection

### ISSUE THREAD

**Issue by dusek** [w3c/aria-practices#772]
The [3.11 Grid pattern](https://w3c.github.io/aria-practices/#grid) discusses in its [WAI-ARIA section](https://w3c.github.io/aria-practices/#grid_roles_states_props) the following item:

> * If the grid supports selection, when a cell or row is selected, the selected element has aria-selected set true.

However the situation where a column is selected is not discussed, even though a keyboard shortcut of `Control + Space` is recommended for column selection action both for layout and data grids.

The reader reading the above-mentioned point is therefore left wondering whether e.g. WAI-ARIA does not support indicating column selection at the moment, or what workaround (e.g. set `aria-selected=true` on all cells in the selected column), if any, is or is not advised.

**Comment by mcking65** [comment]
H@dusek, thank you for catching this. I have made a fix in commit a0c06a9. If you think the fix is not adequate, feel free to re-open this issue.
